tests/test_keep_output_tags.py

```diff
@@ -0,0 +1,39 @@
+from copy import deepcopy
+import re
+
+import nbformat
+import pytest
+
+from nbstripout._utils import strip_output, NBStripoutError
+
+
+@pytest.fixture
+def orig_nb():
+    return nbformat.read('test_keep_output_tags.ipynb', nbformat.NO_CONVERT)
+
+
+@pytest.fixture
+def nb_with_exception():
+    return nbformat.read('test_keep_output_tags_exception.ipynb', nbformat.NO_CONVERT)
```

Comment:
Make sure your read this from the current directory, since tests are usually run with `python setup.py test`, which currently fails.